config/locales/ar.yml

```diff
@@ -575,23 +574,19 @@ ar:
       access_ended_description: You have only limited access to the course now. You are allowed preview the content but cannot complete any target.
       access_ended_title: Preview/Limited Access
       course_complete_description: You've completed our coursework. Feel free to complete targets that you might have left out, and read up at attached links.
-      course_complete_title: Congratulations! You have completed all milestone targets in the final level.
+      course_complete_title: Congratulations! You have completed all milestones in the final level.
       course_ended_description: The course has ended and submissions are disabled for all targets!
       course_ended_title: Course Ended
       level_up_blocked:
         body_middle_pending_review: "are still pending review by a coach. "
         body_middle_rejected: "have been rejected. "
-        body_prefix: "You have submitted all milestone targets in level %{number}, but one or more submissions "
-        body_suffix: You need a passing grade on all milestone targets to level up.
+        body_prefix: "You have submitted all milestones in level %{number}, but one or more submissions "
+        body_suffix: You need a passing grade on all milestones to level up.
```

Comment:
**Unrelated:** I'm pretty sure we don't use these keys anywhere. Levelling up isn't a thing any more. It's likely this file is just outdated.